src/number/types/Fixed18.sol

```diff
@@ -12,6 +12,7 @@ type Fixed18 is int256;
 using Fixed18Lib for Fixed18 global;
 
 using {
+    neg as -,
```

Comment:
Surprised unary minus doesn't conflict with the subtraction overload, but it works.

Reply:
Yeah it's actually supported feature, but quite hidden: https://soliditylang.org/blog/2023/02/22/user-defined-operators/.

![image](https://github.com/user-attachments/assets/62c470d6-2548-46fd-b5ed-cbcd84db3e39)